Example/SwiftSyft/HomeViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  HomeViewController.swift
+//  SwiftSyft
+//
+//  Created by SashaBataieva on 01/20/2020.
+//  Copyright (c) 2020 mjjimenez. All rights reserved.
+//
+
+import UIKit
+
+enum StaticHomeScreenStrings {
+    static let headerDescription = "syft.js/grid.js testing"
+    static let description = """
+            This is a demo using @SwiftSyft@ from @OpenMined@ to execute a multi-worker protocol hosted on grid.js
+    """
+    static let swiftSyft = "SwiftSyft"
+    static let swiftSyftKey = "@SwiftSyft@"
+    static let swiftSyftURL = "https://github.com/OpenMined/SwiftSyft"
+    static let openMined = "OpenMined"
+    static let openMinedKey = "@OpenMined@"
+    static let openMinedURL = "https://github.com/OpenMined/grid.js"
+    static let socketURL = "ws://localhost:3000"
+    static let protocolID = "50801316202"
+    static let connectButtonText = "Connect to grid.js server"
+    static let messagePlaceholder = "Enter your message ..."
+    static let sendButtonText = "Send message"
+}
+
+class HomeViewController: UIViewController, UITextViewDelegate {
+    var isConnected = false
+
+    @IBOutlet weak var headerDescriptionLabel: UILabel!
+    @IBOutlet weak var descriptionTextView: UITextView!
+    @IBOutlet weak var socketURLTextField: UITextField!
+    @IBOutlet weak var protocolIDTextField: UITextField!
+    @IBOutlet weak var connectButton: UIButton!
+    @IBOutlet weak var messageTextView: UITextView!
+    @IBOutlet weak var sendButton: UIButton!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        headerDescriptionLabel.text = StaticHomeScreenStrings.headerDescription
+
+        descriptionTextView.text = StaticHomeScreenStrings.description
+        descriptionTextView.attributedText = descriptionTextView.attributedText?
+            .fillInLink(StaticHomeScreenStrings.swiftSyftKey, with: StaticHomeScreenStrings.swiftSyft, url: StaticHomeScreenStrings.swiftSyftURL)
+            .fillInLink(StaticHomeScreenStrings.openMinedKey, with: StaticHomeScreenStrings.openMined, url: StaticHomeScreenStrings.openMinedURL)
```

Comment:
This is minor, but there's a line length violation triggered on these lines on swiftlint. You can see the warnings swiftlint generates after you build and warnings will appear on the issue navigator of Xcode (Command + 5). You can just create a newline after the parameters like this:

```
            .fillInLink(StaticHomeScreenStrings.swiftSyftKey, with: StaticHomeScreenStrings.swiftSyft, 
                                                               url: StaticHomeScreenStrings.swiftSyftURL)
```

Swiftlint line length violations can be annoying so we can review if you guys want to modify the minimum line length in the future.

Reply:
ok, I will test the keyboard behavior on a device and will fix it. @mjjimenez Thank you :)